### PR TITLE
Documentation update for resampler relocation and removed relative path imports

### LINKED ISSUE #137: documentation for importing wsinterp from parsers error
### Problem
Screenshot 2024-10-26 at 4 21 17 PM

The documentation says to import `wsinterp` from `diffpy.utils.parser` instead of `diffpy.utils.parser.resample` which at first made me think there was a bug for importing `wsinterp`

### PR DESCRIPTION
**Pull request by alisnwu**
closes #137 

@bobleesj could you check this PR before I ping Simon, closes the last issue in 3.5.1

### CODE REVIEW

**Comment by bobleesj** [comment]
@alisnwu Thanks! First, just to double check, are the modifications generated dynamically using the API workflow under `cookiecutter`? If not, let's use that and please also attach a screenshot here to ensure that it works after you've done 

```
cd doc && make html && open build/html/index.html && cd ..
```

For the example, have you followed the instructions that you've tried?

File: doc/source/conf.py [hunk 1]
```diff
@@ -18,6 +18,10 @@
 from importlib.metadata import version
 from pathlib import Path
 
+autodoc_mock_imports = [
+    "diffpy.utils.wx",
```
Comment (bobleesj) [inline doc/source/conf.py:22]: what was this added?

Comment (alisnwu) [reply doc/source/conf.py:22]: sphinx gave an error `WARNING: autodoc: failed to import module 'gridutils' from module 'diffpy.utils.wx'; the following exception was raised:
No module named 'wx' [autodoc.import_object]`
So I followed the cookiecutter instructions for that error

**Comment by bobleesj** [comment]
@alisnwu this PR isn't ready for review right? Please let us know since it's in my notification box. Please make this PR title is descriptive so that we get a sense of what is being worked on from the title alone. Please use the PR/commit guide provided earlier by me.

**Comment by alisnwu** [comment]
No sorry about that, I'll also close this and reopen after issues are fixed

**Pull request closed without merging**